### PR TITLE
Fix web request

### PR DESCRIPTION
**Pull request by Denao2**
Changed the form of the web request to work with latest Sonoff RF bridge with tasmota.
This is what I needed to make the get method work.

### CODE REVIEW

File: yfmos.py [hunk 1]
```diff
@@ -371,7 +371,7 @@ def __exec_B0(self, hostArg, b0, config, profile):
 
         buffer = BytesIO()
         c = pycurl.Curl()
-        url = '%s/cm?cmnd=Backlog%%20%s' % (host, b0.replace(' ', '%20'))
+        url = '%s/cm?cmnd=Backlog%%20%s%%3B%%20RfRaw%%200' % (host, b0.replace(' ', '%20'))
```
Comment (sehaas) [inline yfmos.py]: Make this an option in the configuration file

**Review by sehaas — CHANGES_REQUESTED** [review]
Please make this an option in the configuration file. I'm running my bridge in RAW mode. `RfRaw 0` would disable it every time.

File: yfmos.py (hunk 1 above)
Comment (Denao2) [reply yfmos.py]: Ok. Does the rest of the URL change work for you? Assume default should be without the RfRaw 0 to preserve original behaviour?

I am also experiencing an issue using my modified code. Once the bridge has been running for a while issuing a command causes the bridge to emit a continuous beep that must be resolved by reset. After a restart the commands work fine with no beep.

Comment (sehaas) [reply yfmos.py]: Yes, the new endpoint without `RfRaw 0` is fine.

I also have this beep/reboot problem sometimes, but haven't investigated it any further. The problem could be an invalid RfRaw command or a bug in the [firmware](https://github.com/Portisch/RF-Bridge-EFM8BB1).

Comment (Denao2) [reply yfmos.py]: I have raised an issue on the Portisch firmware project here: https://github.com/Portisch/RF-Bridge-EFM8BB1/issues/146#issue-549824012

**Comment by sehaas** [comment]
If it's ok for you, I'll cherry-pick your first commit (fixing the endpoint) and close this issue. You can submit another PR addressing the optional RfRaw command when it's ready.

**Comment by Denao2** [comment]
I have removed the second commit. As requested I will raise a new PR with the RFRaw 0 as an optional addition once I have had a chance to do so.

**Pull request merged**